viam_cartographer.go

```diff
@@ -121,11 +123,7 @@ func initSensorProcesses(cancelCtx context.Context, cartoSvc *CartographerServic
 	spConfig := sensorprocess.Config{
 		CartoFacade:              cartoSvc.cartofacade,
 		Lidar:                    cartoSvc.lidar,
-		LidarName:                cartoSvc.lidar.Name(),
-		LidarDataFrequencyHz:     cartoSvc.lidar.DataFrequencyHz(),
 		IMU:                      cartoSvc.imu,
-		IMUName:                  cartoSvc.imu.Name(),
-		IMUDataFrequencyHz:       cartoSvc.imu.DataFrequencyHz(),
```

Comment:
Refactoring: `cartoSvc.lidar` already contains the name and data frequency information.